ShopNow/UserApp/views.py

```diff
@@ -86,26 +118,43 @@ def get(self,request,pk=None):
             user=Webuser.objects.all()
             serialized=WebUserSerializer(user,many=True)
             print("This is user: ",request.user)
-            return Response({'user':serialized.data})
-            
-        
+            # try:
+            #     order_serialized=order_serialized(user.user_orders.all(),many=True)
+            #     print(".......",order_serialized.data,"............")
+                
+            #     response_to_Pass={'users_orders':order_serialized.data,'user_info':serialized.data}
+            return Response(serialized.data,status=status.HTTP_200_OK)
```

Comment:
Could you explain why status code iss 200 for pk is equal to none case, its 200 for else case also.